samples/SocketsSample/Startup.cs

```diff
@@ -16,18 +16,17 @@ public class Startup
         // For more information on how to configure your application, visit http://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddRouting();
-
             services.AddSingleton<ProtobufInvocationAdapter>();
             services.AddSingleton<LineInvocationAdapter>();
 
-            services.AddSignalR()
-                    .AddSignalROptions(options =>
+            services.AddSockets();
```

Comment:
Yes especially if they need to configure options for sockets (which there will inevitably be).